### ISSUE TITLE
turn off saving log files by default

### ISSUE THREAD

**Issue by naturallymitchell** [patchworkuniversity/torchbear#235]
https://github.com/foundpatterns/torchbear/issues/162

@dariusc93 could you please take this one

**Comment by dariusc93** [comment]
Yes. I have one already done that I used locally. When I get to my computer in a few I can rebase and push it
